pkg/agent/agent.go

```diff
@@ -280,9 +288,17 @@ func (i *Initializer) initInterfaceStore() error {
 			case interfacestore.AntreaTunnel:
 				intf = parseTunnelInterfaceFunc(port, ovsPort)
 			case interfacestore.AntreaHost:
-				// Not load the host interface, because it is configured on the OVS bridge port, and we don't need a
-				// specific interface in the interfaceStore.
-				intf = nil
+				if port.Name == i.ovsBridge {
+					// Not load the host interface, because it is configured on the OVS bridge port, and we don't need a
```

Comment:
By now, OVS bridge port is mainly used in container case ( for Windows or Antrea IPAM), and no consumer on the OVS bridge interfaceConfig exists, instead, we always use `AdapterNetConfig` to save the uplink configuration in `NodeConfig`. So we didn't load it to any InterfaceConfig. 

For VM case, OVS bridge doesn't  have an OVS port or interface, even mapping uplink. 

Since we want to unify the implementation on AntreaIPAM/Windows/VM Support, I plan to remove the OVS bridge port in a future patch, then this logic should be removed.

Reply:
I feel even today, we need not to set `AntreaHost` to the bridge port?

Reply:
No. If we don't set any interfaceType on the OVS bridge port, the logic would think go into the original logic ( not setting OVS port type), and in this case we can not differentiate the container port or OVS bridge port because there is no identifiers. Then the OVS bridge port is thought as container InterfaceConfig added into the interface store by this line:
```
			switch {
			...
			default:
				// The port should be for a container interface.
				intf = cniserver.ParseOVSPortInterfaceConfig(port, ovsPort, true)
				antreaIFType = interfacestore.AntreaContainer
			}
```

Reply:
I thought we can just use bridge name to recognize the bridge port. At least it should not be set with AntreaHost, esp. after we change host interface to a separate internal port.

Reply:
Do you mean change the logic to identify Bridge port via name in this change? After we change host interface to a separate internal port, we do not need Bridge port any more, so it should not exist.

Reply:
> We can change later. To support upgrade, we may still need to identify and ignore bridge port?

Yes, you are correct